documentation/working-with-testcafe-studio/recording-tests/test-actions/switch-frames.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: docs
+title: Switch Frames
+permalink: /documentation/working-with-testcafe-studio/recording-tests/test-actions/switch-frames.html
+---
+# Switch Frames
+
+The **Switch Frames** category contains actions that allows you to switch the test's [browsing context](https://html.spec.whatwg.org/multipage/browsers.html#windows) to iframes or back to the main window.
+
+* ![Action icon](../../../../images/actions/action-switch-to-iframe-icon.svg) [Switch Context to Iframe](#switch-context-to-iframe)
+* ![Action icon](../../../../images/actions/action-switch-to-main-window-icon.svg) [Switch Context to Main Window](#switch-context-to-main-window)
+
+## Switch Context to Iframe
+
+When you add the **Switch Context to iframe** action to a test, TestCafe Studio switches the test's browsing context to a specified iframe and performs the following actions in this iframe until reaches the [Switch Context to Main Window](#switch-to-main-window) action in the test.
```

Comment:
Here and in the next section:

> When you add,.. TestCafe Studio switches...

Literally, it means that the context is switched when you add an action. While it is switched when the action is executed. i.e. the action switches the context.

```
The **Switch Context to iframe** action switches the test's browsing context to a specified iframe. After this action is executed, TestCafe Studio performs the following actions in this iframe until it reaches the [Switch Context to Main Window](#switch-to-main-window) action in the test.
```